src/protean/core/entity.py

```diff
@@ -138,6 +138,8 @@ def __init__(self, entity_cls_name: str, page: int = 1, per_page: int = 10,
         self._filters = filters
 
         self._result_cache = None
+        self._restructured_filters = {}
```

Comment:
We don't really need to save this in the object, if we ever need it we can just call the function to get it back